src/resources/js/commerce/gateway/stripe/checkout.js

```diff
@@ -604,6 +668,7 @@ tribe.tickets.commerce.gateway.stripe.checkout = {};
 	 * @since 5.3.0
 	 */
 	obj.bindEvents = () => {
+		$( obj.selectors.renderButton ).on( 'click', obj.renderPayment );
```

Comment:
I know that this is mimicking the line below it, however, it is good practice to bind to the document or a container in the event that the item is injected into the page via lazy loading.

```suggestion
		$( document ).on( 'click', obj.selectors.renderButton, obj.renderPayment );
```